bundles/org.openhab.binding.bondhome/src/main/resources/OH-INF/thing/Channels.xml

```diff
@@ -0,0 +1,302 @@
+			light.</description>
+		<category>Light</category>
+		<tags>
+			<tag>Lighting</tag>
+			<tag>Switchable</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="downLightBrightnessChannelType" advanced="false">
+		<item-type>Dimmer</item-type>
+		<label>Down Light Brightness</label>
+		<description>Adjusts the brightness of the fan down light</description>
+		<category>Light</category>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="downLightDimmerStartStopChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Start Dimming Down Light</label>
+		<description>Starts or stops changing the brightness of the ceiling fan down light. The direction of dimming cannot be
+			controlled.</description>
+		<category>Light</category>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="downLightDimmerIncrChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Start Increasing Down Light Brightness</label>
+		<description>Starts or stops increasing the brightness of the ceiling fan down light.</description>
+		<category>Light</category>
+	</channel-type>
+
+	<channel-type id="downLightDimmerDcrChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Start Decreasing Down Light Brightness</label>
+		<description>Starts or stops decreasing the brightness of the ceiling fan down light.</description>
+		<category>Light</category>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="stopChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Stop</label>
+		<description>Stops any ongoing change actions</description>
+		<category>Switch</category>
+		<tags>
+			<tag>Switchable</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="flameChannelType" advanced="false">
+		<item-type>Dimmer</item-type>
+		<label>Flame Level</label>
+		<description>Turns on or adjust the flame level</description>
+		<category>Heating</category>
+	</channel-type>
+
+	<channel-type id="fpFanPowerChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Fireplace Fan State</label>
+		<description>Turns the fireplace fan on or off</description>
+		<category>Heating</category>
+	</channel-type>
+
+	<channel-type id="fpFanSpeedChannelType" advanced="false">
+		<item-type>Dimmer</item-type>
+		<label>Fireplace Fan Speed</label>
+		<description>Adjusts the speed of the fireplace fan</description>
+		<category>Heating</category>
+	</channel-type>
+
+	<channel-type id="openShadeChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Open</label>
+		<description>Opens or closes motorize shades</description>
+		<category>Rollershutter</category>
+	</channel-type>
+
+	<channel-type id="holdChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Hold</label>
+		<description>Tells a shade to stop moving</description>
+		<category>Rollershutter</category>
+	</channel-type>
```

Comment:
For a shade I think it should be modeled as `<item-type>Rollershutter</item-type>` Then it can accept commands: `UpDown`, `StopMove`. See https://www.openhab.org/docs/configuration/items.html#type